src/hls-adapter.js

```diff
@@ -449,6 +449,25 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
     });
     HlsAdapter._logger.debug('Audio track changed', audioTrack);
     this._onTrackChanged(audioTrack);
+    this._handleWaitingUponAudioTrackSwitch();
+  }
+
+  /**
+   * Trigger a playing event whenever an audio track is changed & time_update event is fired.
+   * This align Edge and IE behaviour to other browsers. When an audio track changed in IE & Edge, they trigger
+   * waiting event but not playing event.
+   * @returns {void}
+   * @private
+   */
+  _handleWaitingUponAudioTrackSwitch(): void {
+    const affectedBrowsers = ['IE', 'Edge'];
+    if (affectedBrowsers.includes(Env.browser.name)) {
+      const timeUpdateListener = () => {
+        this._videoElement.dispatchEvent(new Event(BaseMediaSourceAdapter.Html5Events.PLAYING));
+        this._videoElement.removeEventListener(BaseMediaSourceAdapter.Html5Events.TIME_UPDATE, timeUpdateListener);
+      }
+      this._videoElement.addEventListener(BaseMediaSourceAdapter.Html5Events.TIME_UPDATE, timeUpdateListener)
```

Comment:
@odedhutzler we dont have once? (listen only once to the event?)
all in all looks good!